guides/v2.2/cloud/env/variables-deploy.md

```diff
@@ -129,7 +129,10 @@ stage:
 -  **Default**—_Not set_
 -  **Version**—Magento 2.1.4 and later
 
-Use the Project Web UI to set this value. When you move the database from one environment to another without an installation process, you need the corresponding cryptographic information. Magento uses the encryption key value set in the Web UI as the `crypt/key` value in the `env.php` file. This does not overwrite an existing encryption key value in the `env.php` file.
+{{ .bs-callout-warning }}
+You must set the `CRYPT_KEY` value through the Project Web UI instead of the `.magento.env.yaml` file to avoid exposing the key in the source code repository for your environment. See [Set environment and project variables]{{ page.baseurl }}/cloud/project/project-webint-basic.html#project-conf-env-var). 
```

Comment:
```suggestion
You must set the `CRYPT_KEY` value through the Project Web UI instead of the `.magento.env.yaml` file to avoid exposing the key in the source code repository for your environment. See [Set environment and project variables]{{ page.baseurl }}/cloud/project/project-webint-basic.html#project-conf-env-var).
```